recursion/flatten.test.js

```diff
@@ -0,0 +1,37 @@
+const flatten = require("./flatten");
+
+test("for an already flatten array, input should be equal to output", () => {
+    const arr = [1, 2, 3, 4];
+    const flattenArray = flatten(arr);
```

Comment:
expect(arr).toEquall(flatten(arr))
[https://jestjs.io/docs/en/expect#toequalvalue](https://jestjs.io/docs/en/expect#toequalvalue) this equates recursively and should work for comparing arrays.